web/src/features/time/HistoricalTimeHeader.tsx

```diff
@@ -11,61 +11,39 @@ import { RouteParameters } from 'types';
 import { useNavigateWithParameters } from 'utils/helpers';
 import { endDatetimeAtom, isHourlyAtom, startDatetimeAtom } from 'utils/state/atoms';
 
-function setToMidnightUTC(date: Date): Date {
-  const newDate = new Date(date);
-  newDate.setUTCHours(0, 0, 0, 0);
-  return newDate;
-}
-function formatDateToISO(date: Date): string {
-  return date.toISOString().slice(0, -5) + 'Z';
-}
+const TWENTY_FOUR_HOURS = 24 * 60 * 60 * 1000;
```

Comment:
Can we add a comment that specifies that this is in milliseconds? Or rename it to `TWENTY_FOUR_HOURS_MS`?

Reply:
We can but this feels rather nitty and it currently follows our pattern in the api/helpers.ts